### PR TITLE
JsonDeserializer Type Resolution Improvements - javadoc, minor changes

### PR DESCRIPTION
**Pull request by tfactor2**
#1507 
@pivotal-issuemaster This is an Obvious Fix

### CODE REVIEW

File: spring-kafka/src/main/java/org/springframework/kafka/support/serializer/JsonDeserializer.java [hunk 1]
```diff
@@ -97,12 +97,14 @@
 	public static final String USE_TYPE_INFO_HEADERS = "spring.json.use.type.headers";
 
 	/**
-	 * A method name to determine the {@link JavaType} to deserialize the key to.
+	 * A method name to determine the {@link JavaType} to deserialize the key to:
+	 * 'com.Foo.deserialize'. See {@link JsonTypeResolver#resolveType} for the signature.
```
Comment (artembilan) [inline spring-kafka/src/main/java/org/springframework/kafka/support/serializer/JsonDeserializer.java:101]: Can we avoid using `foo/bar` in the production code and docs?
Thanks

Comment (tfactor2) [reply spring-kafka/src/main/java/org/springframework/kafka/support/serializer/JsonDeserializer.java:101]: Thought that it's used intentionally in the project - see `TYPE_MAPPINGS` above:
```java
	/**
	 * Kafka config property to add type mappings to the type mapper:
	 * 'foo=com.Foo,bar=com.Bar'.
	 */
```

**Review by artembilan — APPROVED** [review]
OK. We accept your fix.
Sorry for delay and thank you for contribution; looking forward for more!

**Comment by tfactor2** [comment]
Not a problem, thanks.

**Pull request merged**